test/Interop/Cxx/operators/Inputs/member-inline.h

```diff
@@ -477,4 +477,24 @@ struct HasOperatorCallWithDefaultArg {
   int operator()(int x = 0) const { return value + x; }
 };
 
+struct ClassWithOperatorStarAvailable {
+
+  mutable int value;
+
+  int *operator->() const { return &value; }
```

Comment:
We don't import `operator->` into Swift, so this decl is ignored.
(It's OK to keep it in the test if you'd like, just FYI)